account_invoice_inter_company/models/account_move.py

```diff
@@ -298,11 +302,15 @@ def _prepare_account_move_line(self, dest_move, dest_company):
         self.ensure_one()
         # Use test.Form() class to trigger propper onchanges on the line
         product = self.product_id or False
-        dest_form = Form(
+        dest_form = form or Form(
             dest_move.with_company(dest_company.id),
             "account_invoice_inter_company.view_move_form",
         )
-        with dest_form.invoice_line_ids.new() as line_form:
+        if dest_form.invoice_line_ids:
+            loop = dest_form.invoice_line_ids.edit(0)
```

Comment:
Remove it instead of editing it for avoiding side effects.

Reply:
@igallart just a gentle remind :)

Reply:
Hello, I will not be available for a few weeks, so any change you consider for me is not a problem. @pedrobaeza , for ignorance my, what kind of problems occur?

Reply:
You can accumulate undesired changes in this line done by previous onchanges.

Reply:
Thanks you, I will change it on my return.

Reply:
@pedrobaeza is it ok now?